### ISSUE TITLE
Occasional integration test failures

### ISSUE THREAD

**Issue by padamstx** [IBM/platform-services-python-sdk#10]
**Describe the bug**
Occasionally, the IAM Access Groups integration test will fail in the travis build while trying to remove a member from a group.

**To Reproduce**
This issue typically occurs in the travis build.

**Expected behavior**
The int test runs clean in each parallel build

**Additional context**
The problem most likely occurs due to the fact that our python build is actually 4 builds in parallel (one for each python version 3.5-3.8)

**Comment by padamstx** [comment]
This is  being tracked in a separate issue
